samples/Authorization/audit-event-hub-autorization/README.md

```diff
@@ -0,0 +1,26 @@
+# Audit authorization rules on Event Hub namespaces
```

Comment:
This may be a better fit for the EventHub folder, since the resource type is Microsoft.EventHub